_includes/specials/taproot/en/13-signet.md

```diff
@@ -0,0 +1,132 @@
+  "warning": "Wallet is an experimental descriptor wallet"
+}
+```
+
+To test the wallet, you'll need to request funds from the signet faucet,
+which currently does not support taproot bech32m addresses, so we'll
+first create a bech32 address.
+
+```text
+$ bitcoin-cli -named -signet getnewaddress address_type=bech32
+tb1q7ch3ef34rqu3kyp434s8uq80jdgn04x0c9r80g
+```
+
+Enter this address on the [signet faucet][] request page and wait a few
+moments for the funds to be sent.  You'll then need to wait for a
+confirmation, which will take the same variable amount of time you'd
+expect on mainnet (typically up to 30 minutes, but sometimes longer).
+After you have a confirmation, you can create a taproot bech32m address
+and send your funds there.
+
+```text
+$ bitcoin-cli -named -signet getnewaddress address_type=bech32m
+tb1pyk0ywqynuq34y8dkn4z8gqcqh4z6wvd9hmsa0h4ecd6g28yxu8dsah4ajv
+
+$ bitcoin-cli -named -signet sendtoaddress address=tb1pyk0ywqynuq34y8dkn4z8gqcqh4z6wvd9hmsa0h4ecd6g28yxu8dsah4ajv amount=0.0001
+bd6dbd2271a95bce8a806288a751a33fc4cf2c336e52a5b98a5ded432229b6f8
+```
+
+If you look at the transaction, you'll notice both the P2TR script whose
+address you paid plus a second P2TR script that Bitcoin Core is using
+for the change.
+
+```text
+$ bitcoin-cli -signet getrawtransaction bd6dbd2271a95bce8a806288a751a33fc4cf2c336e52a5b98a5ded432229b6f8 true | jq .vout
+"vout": [
+  {
+    "value": 0.00089786,
+    "n": 0,
+    "scriptPubKey": {
+      "asm": "1 2b8d255ab42866f4862051dea1d4ca3b13f84daa96fb76511cd2cc8e8c92ea99",
+      "hex": "51202b8d255ab42866f4862051dea1d4ca3b13f84daa96fb76511cd2cc8e8c92ea99",
+      "address": "tb1p9wxj2k459pn0fp3q2802r4x28vflsnd2jmahv5gu6txgaryja2vss3lua2",
+      "type": "witness_v1_taproot"
+    }
+  },
+  {
+    "value": 0.00010000,
+    "n": 1,
+    "scriptPubKey": {
+      "asm": "1 259e470093e023521db69d44740300bd45a731a5bee1d7deb9c374851c86e1db",
+      "hex": "5120259e470093e023521db69d44740300bd45a731a5bee1d7deb9c374851c86e1db",
+      "address": "tb1pyk0ywqynuq34y8dkn4z8gqcqh4z6wvd9hmsa0h4ecd6g28yxu8dsah4ajv",
+      "type": "witness_v1_taproot"
+    }
+  }
+]
+```
+
+If you sent to your own wallet software in the step above, you'll
+probably want to wait at least one confirmation before spending back to
+Bitcoin Core.  Since we're using Bitcoin Core for both sides here, we
+can send the money to a second bech32m address for testing immediately:
+
+```text
+$ bitcoin-cli -named -signet getnewaddress address_type=bech32m
+tb1p53qvqxja52ge4a7dlcng6qsqggdd85fydxs4f5s3s4ndd2yrn6ns0r6uhx
+$ bitcoin-cli -named -signet sendtoaddress address=tb1p53qvqxja52ge4a7dlcng6qsqggdd85fydxs4f5s3s4ndd2yrn6ns0r6uhx amount=0.00099
+24083fdac05edc9dbe0bb836272601c8893e705a2b046f97193550a30d880a0c
+```
+
+For this spend, we can look at one of the inputs and see that its
+witness contains nothing but a single 64-byte signature, which is
+[smaller][p2tr vs p2wpkh] than the pubkey and longer signature that
```

Comment:
This sentence is a bit long. By the time I got to the "and" between "the pubkey" and "longer signature" I'd lost the structure a bit, and on first parse thought that it was joining two clauses in the sentence rather than joining "pubkey" and "signature". 

How about breaking things up a bit like:

"For this spend, we can look at one of the inputs and see that its witness contains nothing but a single 64-byte signature. That's smaller than the witness which would’ve been required if this was a P2WPKH spend or any other type of older Bitcoin spend, which contain both a pubkey and a (longer) signature."

Reply:
LGTM